symfony/composer.json

```diff
@@ -27,8 +30,6 @@
             "OrangeHRM\\ORM\\": "./lib/orm",
```

Comment:
@samanthajayasinghe Psr4 autoload going to define composer, but seems we have to restructure directories within the plugins, to adhere Psr4 convention. Since previously all classes loaded in root namespace.
We can move plugin related autoloading into plugin configurations. Keep core things in composer.

Having some doubt with runtime Psr4 autoloading, since it may not optimize when we use optimized composer autoloading

Reply:
yeh runtime Psr4 autoloading may not work since class-map needs to be cache.